run_test.sh

```diff
@@ -26,7 +26,8 @@ function run_npm_test {
 # Run linter on client side code
 function run_npm_lint {
   cd static
-  npm run lint && npm run tslint
```

Comment:
is npm run lint now equivalent to tslint?

also, i wonder if we should just add --fix, since that will be what each developer has to do right after anyway..

btw, can we speed up npm install? it really slows things down if we have to run it every time. some options:
- add a separate option to only run npm install
- npm list eslint || npm install eslint
- https://www.npmjs.com/package/npm-cache

Reply:
Yep, since this is specific to eslint. I use the option of "npm list eslint || npm install eslint"